flags/src/main/java/com/yahoo/vespa/flags/custom/NodeMaintainerDurations.java

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019 Oath Inc. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+package com.yahoo.vespa.flags.custom;
+
+import com.fasterxml.jackson.annotation.JsonAnyGetter;
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+import java.time.Duration;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+
+/**
+ * {@code NodeRepositoryMaintenance} depends on various durations, e.g. the reboot interval. This class
+ * defines a serialization format for specifying such durations as a JSON object with keys matching
+ * the name (reboot_interval) and with the value being the duration in second as an integer.
```

Comment:
Let the names be the same as class names? That's what we use to enable/disable maintainers.

Reply:
The current names are fairly descriptive, while attaching a duration to a Maintainer's subclass name may not be. Also, a Maintainer's interval may be different from the duration controlled here - for instance making smaller adjustments to fail_grace in main would typically not affect the NodeFailer's Maintainer interval.

Reply:
NA